src/redux/categories/categories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const Categories = () => {
+  const QuotesData = {
+    name: 'UNDER',
+    quote:
+            'CONSTRUCTION',
+  };
+
+  const Category = () => {
+    const { name, quote } = QuotesData;
+    return (
+      <>
+        <div className="content">
+          <div className="nameQuote">
+            {name}
+          </div>
+          <div className="quote">
+            {quote}
+          </div>
+        </div>
+      </>
+    );
+  };
+
+  return (
+    <Category />
+  );
+};
+
+export default Categories;
```

Comment:
- [ ] The **`redux`** directory is meant for redux-related configurations, not components. At this point in the project, this file should be empty. Kindly move this **Categories** component into a new file in the `src/component` folder as that is where your components should live.